src/view/ChewingEditor.cpp

```diff
@@ -178,6 +178,27 @@ void ChewingEditor::showAbout()
     aboutBox.exec();
 }
 
+void ChewingEditor::showDeleteConfirmWindow()
+{
+    QString text = QString("Do you want to delete this phrase?");
+
+    QMessageBox deleteBox(this);
```

Comment:
need to add a icon for this dialog? or it's too distracting?
https://doc.qt.io/qt-5/qmessagebox.html#Icon-enum